projects/hyperdash-angular/src/editor/model-editor.service.test.ts

```diff
@@ -17,8 +17,12 @@ describe('Model editor service', () => {
 
   beforeEach(() => {
     mockEditorApiFactory = {
-      buildLeafEditorApi: jest.fn((_, editorData: LeafEditorData) => `${editorData.title} LEAF API`) as jest.Mock,
-      buildNestedEditorApi: jest.fn((_, data: UnresolvedCompositeEditorData) => `${data.title} NESTED API`) as jest.Mock
+      buildLeafEditorApi: jest.fn((_, editorData: LeafEditorData) => {
```

Comment:
What was wrong here? We're changing to ban one line functions?

Reply:
```
        "arrow-body-style": ["error", "never"],
```
I think I will have to update this rule with the above value and then fix a bunch of errors. I think I should make this change. What do you think? I created a separate PR so we can go over the rules too.

Reply:
I would rather we leave this with the default behavior which is what our previous linter was using - `arrow-body-style: ["error", "as-needed"]`

Reply:
Sure. I will also compare old tslint with new rules. and call out any changes.

Reply:
ok. updated. Please check. 

I kept all the disabled rule towards the end of eslintrc. Made a couple of minor eslint fixes like nullish-coalescing.